.circleci/config.yml

```diff
@@ -14,5 +14,5 @@ jobs:
           paths:
             - ~/.m2
           key: v1-dependencies-{{ checksum "pom.xml" }}
-      - run: mvn install -Pqulice -Pitcase
+      - run: mvn install -Pqulice
```

Comment:
@olenagerasimova why did you remove it?

Reply:
@victornoel build failed, smth wrong with infrastructure

Reply:
@olenagerasimova I've created an issue #52 for this then, thx